data/avgi/avgi weapons.txt

```diff
@@ -0,0 +1,804 @@
+	description "Snapdragon storage"
+
+
+outfit "Snapdragon Launcher"
+	category "Secondary Weapons"
+	licenses
+		"Avgi Atomics"
+	cost 1370000
+	thumbnail "outfit/casaba howitzer launcher"
+	"mass" 24
+	"outfit space" -24
+	"weapon capacity" -24
+	"gun ports" -1
+	"snapdragon capacity" 12
+	weapon
+		sprite "projectile/finisher activating"
+			"frame rate" 1
+			"no repeat"
+		sound "finisher"
+		ammo `"Snapdragon" Nuclear Shaped Charge`
+		icon "icon/finisher"
+		"fire effect" "finisher sparks" 8
+		"die effect" "finisher fragment" 6
+		"hit effect" "medium explosion"
+		"inaccuracy" 3
+		"velocity" 6
+		"lifetime" 30
+		"reload" 90
+		"firing energy" 6
+		"firing heat" 3
+		"optical tracking" 0.9
+		"radar tracking" 0.6
+		"missile strength" 36
+		"submunition" "Active Snapdragon"
+	description `Casaba Howitzer`
+
+
+outfit "Active Snapdragon"
+	weapon
+		sprite "projectile/thunderhead"
+			"frame rate" 2.4
+		"live effect" "finisher trail" 50
+		"hit effect" "thermonuclear flash"
+		"hit effect" "prompt radiation" 12
+		"hit effect" "fission fragments" 60
+		"die effect" "thermonuclear flash"
+		"lifetime" 1200
+		"acceleration" 0.6
+		"drag" .03
+		"turn" 6
+		"homing" 3
+		"optical tracking" 0.9
+		"radar tracking" 0.6
+		"split range" 900
+		"missile strength" 36
+		"submunition" "Snapfront" 120
+		"submunition" "Snapback" 90
+
+
+outfit "Snapfront"
+	weapon
+		sprite "effect/plasma impact"
+			"frame rate" 30
+			"no repeat"
+			"scale" 0.5
+		"hit effect" "nuclear spallation"
+		"live effect" "casaba trail" 60
+		"inaccuracy" 9
+			narrow
+		"lifetime" 30
+		"random lifetime" 30
+		"velocity" 60
+		"random velocity" -60
+		"shield damage" 120
+		"hull damage" 180
+		"hit force" 180
+
+
+outfit "Snapback"
+	weapon
+		sprite "effect/composed"
+			"frame rate" 60
+			"no repeat"
+		"hit effect" "nuclear spallation"
+		"live effect" "casaba trail" 60
+		"inaccuracy" 90
+			narrow
+			inverted
+		"random lifetime" 30
+		"random velocity" -60
+		"shield damage" 60
+		"hull damage" 90
+		"hit force" 90
+
+
+effect "casaba trail"
+	sprite "effect/flame"
+		"frame rate" 30
+		"no repeat"
+	"random angle" 30
+	"random lifetime" 12
+	"random velocity" -6
+
+
+outfit "Equinox Torpedo"
+	category "Ammunition"
+	licenses
+		"Avgi Atomics"
+	cost 660000
+	thumbnail "outfit/equinox torpedo"
+	"mass" 3
+	"equinox capacity" -1
+	description "The Equinox makes no compromises with its design goal of fitting the largest thermonuclear warhead feasible. After minimalizing its propulsion system to little more than a few reaction control thrusters, the Equinox is essentially an unguided torpedo. A carrying ship must instead both accelerate and aim the torpedo properly to ensure a hit."
+	description "A thin coating of ablative polymer designed to resist lasers was the only mass the Avgi were willing to allocate to armor, and as such the Equinox is easily shot down by most modern antimissile systems."
+
+
+outfit "Equinox Torpedo Pylon"
```

Comment:
Again, this is a much more interesting and balanced weapon than the Orchid. It's comparable in many ways to human nukes - similar size, and only a single shot. It has a better damage output, but that comes at the cost of any homing ability at all - just an unguided projectile, with a modest speed, and an extreme damage output.
Much like the Snapdragon, these kinds of interesting caveats and tradeoffs are needed to balance out these kinds of damage numbers, but also like the Snapdragon in this case you've succeeded in doing so.
The 7200 range is high, but not excessive, and far less valuable on an unguided weapon than it would be on a guided one in any case - so, unlike the Snapdragon, Orchid and Nettle, the Equinox's range isn't excessive.